src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -58,13 +58,19 @@
           :data-test-id "status-icon" :data-test-value "unknown"}
    (spinner)])
 
-(defn icon-for-cluster-status [status]
```

Comment:
Moved this logic to a method within `NotebooksTable` because icon rendering now depends on some state (e.g. icons may be links).